projects/module-1/hang-in-there.md

```diff
@@ -120,15 +120,15 @@ To earn a given score, an application must meet the requirements listed in that
 * **4:**
   - Team uses a [PR template](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository) consistently
   - Team habitually conducts thorough code reviews in the GitHub GUI to document the progress of the application
-  - Team has sought out code reviews from one or more mentors. Mentors must add their code review to a pull request that can be viewed.
+  - Team has sought out code reviews from one or more mentors. Mentors must add their code review to a pull request that can be viewed. Consider letting your PM know that a mentor has reviewed to ensure they see it.
 
 * **3:**
   - Commits are atomic and frequent, effectively documenting the evolution/progression of the application. Remember, a commit should be one “unit” of work.
   - Commit messages are consistent, descriptive, concise and begin with a verb and capital letter
   - Team uses PRs to screen/verify code before adding it to the main branch
   - Branches are consistently used for individual features
   - There is no more than a 10% disparity in code contributions between teammates. Note: this is checked via the “Insights” and “Contributors” tab in your GitHub repo.
-  - README is well formatted and gives good context about the project. At minimum, a sufficient README should contain - contributors, technologies used, instructions for running and viewing the project, deploy link (gh-pages), images/GIFs if necessary, future additions, etc. Think about what a user needs to understand and get the full picture of the application.
+  - README is well formatted and gives good context about the project. At minimum, a sufficient README should contain - links to all contributors' GitHubs, technologies used, instructions for running and viewing the project (as a user, not as a student building the project), deploy link (gh-pages), images/GIFs if necessary, future additions, etc. Think about what a user needs to understand and get the full picture of the application.
```

Comment:
Can we remove the "if necessary" from the gifs/screenshots item? I think they should include a screenshot or gif in every README and I always give that feedback.